### PR TITLE
DHCP scope

### PR DESCRIPTION
**Pull request by DarrylvanderPeijl**
Added the DHCPScope to allow custom DHCP scope.

### CODE REVIEW

File: Scripts/2_CreateParentDisks.ps1 [hunk 1]
```diff
@@ -736,11 +782,11 @@ If (!( $isAdmin )) {
                     xDhcpServerOption MgmtScopeRouterOption
                     {
                         Ensure = 'Present'
-                        ScopeID = '10.0.0.0'
+                        ScopeID = ($DHCPscope+"0")
```
Comment (jaromirk) [inline Scripts/2_CreateParentDisks.ps1]: Is this +"0" needed?

Comment (DarrylvanderPeijl) [reply Scripts/2_CreateParentDisks.ps1]: Yes, in the beginning of the script I strip the last character:

$DHCPscope = $DHCPscope.Substring(0,$DHCPscope.Length-1) 

So on line 785 "$DHCPscope" equals 10.0.0.

Comment (jaromirk) [reply Scripts/2_CreateParentDisks.ps1]: OK, so it means, that scope has to end with .0 (all /24 do :)

Comment (DarrylvanderPeijl) [reply Scripts/2_CreateParentDisks.ps1]: That is correct. That is the reason why only a /24 is allowed.

**Pull request merged**